docs/how-to/installation/gcp.txt

```diff
@@ -306,6 +314,85 @@ If you deployed with a service account JSON key file, the same
 credentials file will be used for deprovisioning. Otherwise, default
 Google Cloud credentials are used.
 
+.. _gcp-master-yaml-template:
+
+******************************
+ Custom master.yaml templates
```

Comment:
same

Reply:
this file is structured differently, so for better or for worse that seems to be correct here.